azure-simple/terraform/datasources.tf

```diff
@@ -0,0 +1,21 @@
+data "template_file" "bastion_tempalte_script" {
```

Comment:
If possible let's add the license header

Reply:
Done